runtime/src/iree/builtins/ukernel/arch/x86_64/CMakeLists.txt

```diff
@@ -192,7 +192,8 @@ iree_select_compiler_opts(IREE_UK_COPTS_X86_64_AVX512_BASE
 iree_select_compiler_opts(IREE_UK_COPTS_X86_64_AVX512_VNNI_RELATIVE
   CLANG_OR_GCC
     "-mavx512vnni"
-  MSVC
+  CLANG_CL
+    "-mavx512vnni"
```

Comment:
Clang-CL supports such GCC-style flags ?

Reply:
good catch, it silently ignores this one - will remove for clarity

Reply:
I can use ` /clang:-mavx512vnni` but it's not getting passed to check_cxx_source_compiles hmm

Reply:
fixed! neato